### DISCUSSION TITLE
bogue

### DISCUSSION THREAD

**Discussion by DBMC0613** [lllyasviel/Fooocus#3720] [Q&A] [answered]
bonjour, je me retrouve face a 2 problèmes avec fooocus 2.5.5,  les aperçues d'image dans le carré dans l'onglet "styles" ils ne s'affichent pas, et le second problème lorsque je sélectionne un ou plusieurs styles différents pour générer une image,  les styles que j'ai sélectionné ce désélectionnes sans que je n'y touche

**Comment by mashb1t** [comment, marked as answer]
Please ensure that your browser is not blocking any javascript and/or css by checking the network tab. Disable browser extensions or allow loading scripts in your ad blocker if necessary.
